docs/framework/winforms/controls/add-and-remove-columns-in-the-datagrid-using-the-designer.md

```diff
@@ -11,7 +11,7 @@ ms.assetid: 9e709f35-0a8c-4e7e-b4c4-bacb7a834077
 # How to: Add and Remove Columns in the Windows Forms DataGridView Control Using the Designer
 The Windows Forms <xref:System.Windows.Forms.DataGridView> control must contain columns in order to display data. If you plan to populate the control manually, you must add the columns yourself. Alternately, you can bind the control to a data source, which generates and populates the columns automatically. If the data source contains more columns than you want to display, you can remove the unwanted columns.  
   
- The following procedures require a **Windows Application** project with a form containing a <xref:System.Windows.Forms.DataGridView> control. For information about setting up such a project, see [How to: Create a Windows Application Project](https://msdn.microsoft.com/library/b2f93fed-c635-4705-8d0e-cf079a264efa) and [How to: Add Controls to Windows Forms](../../../../docs/framework/winforms/controls/how-to-add-controls-to-windows-forms.md).  
+ The following procedures require a **Windows Application** project with a form containing a <xref:System.Windows.Forms.DataGridView> control. For information about setting up such a project, see [How to: Create a New Windows Application Project](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/42wc9kk5(v=vs.100)) and [How to: Add Controls to Windows Forms](../../../../docs/framework/winforms/controls/how-to-add-controls-to-windows-forms.md).  
```

Comment:
@rpetrusha it's kind of impressive that we don't have any get started article on .NET using VS on docs fresh. The best that exists on docs is this one: https://docs.microsoft.com/en-us/visualstudio/ide/step-1-create-a-windows-forms-application-project which is part of a tutorial, but at least it has more up-to-date instructions.

Do you think we change this to the VS one until we start working on the Winforms content for Core?

Reply:
I agree, @mairaw. The article in previous-versions is badly dated, and it's also much too minimalistic (all you do is create the project, which is fairly intuitive). Even the one at < https://docs.microsoft.com/en-us/visualstudio/ide/step-1-create-a-windows-forms-application-project> doesn't completely correspond to the UI, but at least it's close and is a bit more ambitious. So I agree that it would be a much better link target.

The content on [WinForms and the command line](https://docs.microsoft.com/en-us/dotnet/framework/winforms/how-to-create-a-windows-forms-application-from-the-command-line) still looks serviceable, though it's not completely accurate and somewhat dated.

Clearly, we need to begin thinking about content we'll need for 3.0.